zulipterminal/ui.py

```diff
@@ -109,24 +109,41 @@ def main_window(self) -> Any:
     def keypress(self, size: Tuple[int, int], key: str) -> str:
         if self.controller.editor_mode:
             return self.controller.editor.keypress((size[1],), key)
-        elif key == "w":
+        elif is_command_key('SEARCH_PEOPLE', key):
             # Start User Search if not in editor_mode
             self.users_view.keypress(size, 'w')
             self.body.focus_col = 2
             self.user_search.set_edit_text("")
             self.controller.editor_mode = True
             self.controller.editor = self.user_search
             return key
-        elif key == "q":
+        elif is_command_key('SEARCH_STREAMS', key):
             # jump stream search
             self.left_col_w.keypress(size, 'q')
             self.body.focus_col = 0
             self.stream_w.search_box.set_edit_text("")
             self.controller.editor_mode = True
             self.controller.editor = self.stream_w.search_box
             return key
-        else:
-            return super(View, self).keypress(size, get_key(key))
+        elif is_command_key('HELP', key):
+            # Show help menu
+            self.controller.show_help()
+            return key
+        elif is_command_key('PREVIOUS_MESSAGE', key):
```

Comment:
I think it would help to add a comment on why this replacement needs to be done.